### PR TITLE
Blog posts/on em role at deliveroo

### PR DESCRIPTION
**Pull request by victoria-puscas**
Article about what do EMs do at Deliveroo.

### CODE REVIEW

File: _authors/_example-bio0.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout:     authors
+collection: authors
+name:       "Your Name"
+jobtitle:   "Your Job Title"
+portrait:   "/images/portraits/your-portrait.jpg"
+---
+
+Write a bio here?
```
Comment (kapowaz) [inline _authors/_example-bio0.md]: 👏

File: _posts/2018-03-08-on-em-role-at-deliveroo.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title:  "What does an Engineering Manager do at Deliveroo?"
+authors:
+   - "Victoria Puscas"
+excerpt: >
+  Do many software engineers know what an Engineering Manager(EM) actually does? As I moved through several companies through my career I saw this role in so many different colors and forms. Every time I was wondering - what is this job actually?
```
Comment (kapowaz) [inline _posts/2018-03-08-on-em-role-at-deliveroo.md]: > `Engineering Manager(EM)`

Missing a space between `Manager` and `(EM)`?

File: _posts/2018-03-08-on-em-role-at-deliveroo.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+Because we want to empower engineers to find solutions to problems at the edge of computer science at great scale.
+
+## How are we doing it?
+
+### Hiring
+
+Your first thought might be ‘isn’t that what recruiters do?’.
+It is, but it is EMs responsibility to be actively involved in sourcing - exploring LinkedIn profiles, searching good candidates by asking people around for recommendations, go to relevant events/meetups when appropriate, and anything that could help finding good people. It’s like a treasure hunt. It is also very, very hard work.
+
+ While folks at the big 5s might have an easy job with attracting many candidates, as everyone knows them, here at Deliveroo we’re still working on spreading the word out there. It might also feel frustrating at times, after spending few hours a day finding good profiles, crafting messages for them, be waiting for a reply back. Some of the very few people that will get back might not be interested. And then we go again.
```
Comment (kapowaz) [inline _posts/2018-03-08-on-em-role-at-deliveroo.md]: Might it be worth explaining what the Big Fives are? (and possibly spelling it out that way so people understand what it refers to)?

Comment (kapowaz) [reply _posts/2018-03-08-on-em-role-at-deliveroo.md]: > `crafting messages for them, be waiting for a reply back.` 

Possibly missing ‘to’ before ‘be waiting…’?

File: _posts/2018-03-08-on-em-role-at-deliveroo.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+Once there are people in the pipeline, EMs are occasionally asked to talk to potential candidates over the phone, in order to get them excited about us and also answer questions candidates might have (often really tricky ones).
+
+Interviewing will become part of your weekly routine. It is very important to get good at it - ask good questions, ask follow-up questions so that in the very short hour we could get as much information as it is possible in order to make a good decision - would this person be good at the job? Would they feel comfortable in the engineering team? Could we set them up for success at Deliveroo? Are we certain they will help Deliveroo grow by bringing their energy, experience and ideas?
+
+And finally providing feedback to candidates and make sure people that were involved in the interview are providing good, quality feedback. That means at the end of debrief the recruitment team can hand all the feedback over to the hiring manager and they could make a decision based on what they read. So providing good feedback and giving people feedback on their feedback.
+
+**Feedback is a big thing.** It needs a separate section (below).
```
Comment (kapowaz) [inline _posts/2018-03-08-on-em-role-at-deliveroo.md]: Perhaps say something like “I’ll talk more about this below”? At the moment it reads a little bit like this might be a TODO note to yourself?

File: _posts/2018-03-08-on-em-role-at-deliveroo.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+### Line management
+
+Line management is at the core of what an EM is doing.
+It is hugely important to be able to keep the talent. The EM’s role is to make sure everyone in the team has the skills, space and the right setup to do great things. Deliveroo will be one of the many stops an engineer might have along their career. The EM needs to work with each engineer to ensure they get the most of the time they’re with the company, they (EM’s) help engineers progressing with their professional life.
```
Comment (kapowaz) [inline _posts/2018-03-08-on-em-role-at-deliveroo.md]: Suggested minor edit to the final sentence here:

> The EM needs to work with each engineer to ensure they get the most of the time they’re with the company, and to progress with their professional life.

File: _posts/2018-03-08-on-em-role-at-deliveroo.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
...Ms in the past. Therefore it is very important to set the expectations correctly. The EM’s job is to make sure engineers understand what’s the purpose of the 1:1, and work with engineers to make them useful. These meetings need preparation from both engineer and their EM. Even if it is best to have engineer talking for mostly the whole (if not the whole) session, it is required from the EM to come prepared - know what’s going on with the project overall but also be aware of who’s working on what and have an idea of the pace, progress, blockers, etc, so that the 1:1 is not a status update, unless there’s something urgent which needs escalated with the EM in order to get moving. It is a rare opportunity to have people come and talk openly about problems, concerns or fears they have. Usually the EM will need to pick up on little clues in order to spot issues which are best to be addressed early. Being capable of reading between the lines is a very subtle skill to master in this role.  **Also being supportive rather than directive is something we require EMs do at Deliveroo.** Needless to say it isn’t a good practice to cancel 1:1s if both EM and engineer are in the office/are working.
```
Comment (kapowaz) [inline _posts/2018-03-08-on-em-role-at-deliveroo.md]: This (and the following two) paragraph is pretty large for a bullet point. Might it make more sense to split this list into subheadings so that the text for each can be divided into shorter paragraphs?

File: _posts/2018-03-08-on-em-role-at-deliveroo.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+We are working with the business, working closely with the PM of the team in order to assure there is enough scope for everyone and it is aligned to quarterly goals which are in turn aligned with yearly company goals.
+
+Being aware of what’s going on in other teams, attending product reviews is indispensable in the role. Bringing the necessary information to the team and also letting others know about what is the team working on and what will it be working on in medium and long term is helping engineers to be focussed and efficient at shipping features at good pace.
+
+Sometimes EMs along with PMs act as air traffic controllers in order to make sure all the information, resources are allocated, the business is up to date with the progress of the work and risks are managed properly.
+
+### Innovation
+
+This is ultimately what we’re seeing for in order to be able to find and solve unique problems. This is hard to achieve and it can not be scheduled as a block in a calendar. We need to let engineers innovate and think outside the box and this is only possible in a transparent, friendly and supportive environment.
```
Comment (kapowaz) [inline _posts/2018-03-08-on-em-role-at-deliveroo.md]: > This is ultimately what we’re seeing for …

Should this say `This is ultimately what we’re looking for` ?

File: _posts/2018-03-08-on-em-role-at-deliveroo.md (hunk 3 above)
Comment (victoria-puscas) [reply _posts/2018-03-08-on-em-role-at-deliveroo.md]: On the 5s...I didn't want to do any advertising. But I'm talking about google, fb and such. I'm not sure I want to explicitly have their names... Maybe I should just rephrase it to something like 'at the biggest tech companies' instead of big 5s, would that be better?

Comment (kapowaz) [reply _posts/2018-03-08-on-em-role-at-deliveroo.md]: That sounds good, yeah 😄
